README.md

```diff
@@ -15,25 +15,52 @@ This template can be used for easily setting up a data science or machine learni
+If you don’t have an Azure subscription, create a free account before you begin. 
+Try the [free or paid version of Azure Machine Learning](https://aka.ms/AMLFree) today.
 
 ### 2. Create repository
 
-To get started with ML Ops, simply create a new repo based off this template, by clicking on the green "Use this template" button:
+Create a new repo based off this template, by clicking on the green *'Use this template'* button:
 
 <p align="center">
   <img src="https://help.github.com/assets/images/help/repository/use-this-template-button.png" alt="GitHub Template repository" width="700"/>
 </p>
 
-### 3. Setting up the required secrets
+### 3. Setting up GitHub Secrets & Azure Service Principal Authentication (SP)
 
-A service principal needs to be generated for authentication and getting access to your Azure subscription. We suggest adding a service principal with contributor rights to a new resource group or to the one where you have deployed your existing Azure Machine Learning workspace. Just go to the Azure Portal to find the details of your resource group or workspace. Then start the Cloud CLI or install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) on your computer and execute the following command to generate the required credentials:
+A service principal (SP) needs to be generated for authentication and accessing your Azure subscription. We suggest adding a service principal with contributor rights to a new resource group, or to the one where you have deployed your existing Azure Machine Learning workspace. 
+
+Navigate to the [Azure Portal](https://www.portal.azure.com/) to find the details of your resource group or workspace. Create a Machine Learning workspace if you haven't already. 
+
+**Note:** If you can't add a role assignment, you are not permissioned with the ability to create a service principal. Contact your IT admin to remedy this. *(workspace > 'Access Control(IAM)' > 'Add a role assignment' > **'Add'**)*. 
+
+<p align="center">
+  <img src="/docs/images/workspace-permissions.png" alt="Workspace Permissions" width="550"/>
+</p>
+
+**Service Principal (SP) Authentication using AZ CLI**
+
+[Additional SP setup methods can be found here](https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/manage-azureml-service/authentication-in-azureml/authentication-in-azureml.ipynb
+), in the section titled 'Service Principal Authentication'.
+
+**You will need the following to create your SP in the AZ CLI:**
+
+- **{service-principal-name}** = pick a new name for your service principal (e.g.workspacename_SPauth)
+- **{subscription-id}** = *Workspace > Overview > subscription ID*
+- **{resource-group}**= *Workspace > Overview > resource group*
+
+<p align="center">
+  <img src="/docs/images/workspace-overview.png" alt="Workspace Overview" width="550"/>

```

Comment:
For consistency, could we update the images to make them styled similar to the `docs/images/secrets.png` where the sensitive info is blurred and the callout is shown with a red rectangle.